### ISSUE TITLE
Grouped Updates - pausing is not respected

### ISSUE THREAD

**Issue by klyonrad** [depfu/feedback#33]
Same as title

I hoped that this could have been a way to postpone some gems from updating :/

**Comment by theflow** [comment]
Hmm, yeah, this is going to be quite tricky for us since we basically just run `bundle update`. But putting in a version requirement into your Gemfile should work?

Could you explain your use case a bit more?

**Comment by klyonrad** [comment]
Well that explains it.

To give you some context for evaluating the importance 

Usecase:
* Individual pull requests for the dev and test gems are annoying 
* by doing them with grouped updates we can give them a more relaxed time schedule
* in generel I like the *run out of spec updates* option
* currently I have two gems that I would like to be paused: for one I am waiting on a fix, for the other I need to extra work that I want to postpone (but get rmindet if it gets a new update)


The story is basically: dev and test gems mostly don't break things so if they wouldn't be annoying we could update them every week without even looking. Sometimes they need extra work that is just rarely high prio

**Comment by theflow** [comment]
Hey @klyonrad, we shipped support for this last week!

Paused updates are now respected in all grouped updates except when updating all dependencies in Bundler (as we're just running `bundle update`). We're looking at options there, but I think your use case of updating dev dependencies should fully work now! Let me know if see any issues or have more feedback.

**Comment by klyonrad** [comment]
Wow, thanks a ton, I did not expect this :)
